util/opentitan-pgm-fpga/opentitan-pgm-fpga

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Program an FPGA with a bitstream
+"""
+import argparse
+import os
+import subprocess
+import sys
+
+
+def program_bitstream_with_vivado(part,
+                                  bitstream,
+                                  hw_target="",
+                                  jtag_frequency=""):
```

Comment:
I probably wouldn't have default values here: the one and only callsite supplies all the arguments.

Also, do you need to set up the environment with `JTAG_FREQ` and `HW_TARGET`? At the moment, this function doesn't use the arguments.